src/components/common/CodeReviewContainer.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import CodeReview from './CodeReview';
+import CodeEditor from './CodeEditor';
+
+function CodeReviewContainer({ editorOpen }) {
+  return <>{editorOpen ? <CodeEditor /> : <CodeReview />}</>;
```

Comment:
Do you need to wrap this in `<></>`?

Reply:
Actually no, you are right. I will remove them.

Reply:
Thinking about this and we may have an issue with the height of the iframe. At the moment it is set by `CodeReview` (before it was the top-level component) but now, when we display the `CodeEditor` component i don't think it will be handeled correcly ... 

We could put the `CodeEditor` inside `CodeReview` and have him hide or show the Review or the code editor based on `editorOpen` such that `CodeReview` would hold a ref to it's parent most component and adjust the height based on that ... what do you think @juancarlosfarah

Reply:
Addressed this in b8282b5. Deployed and tested ✅